### PR TITLE
Support evaluation entity in demo data

### PR DESCRIPTION
**Pull request by mtjandra**
Supporting draft evaluation entity in demo data

### CODE REVIEW

File: demo/d2l-evaluation-hub/demo-data-parser.js [hunk 1]
```diff
@@ -263,15 +263,16 @@ function parseActivities(data, users, activityNames, courses) {
...
+			isDraft: row.isDraft
 		};
 	});
 
 	return parsedActivities;
 }
 
 function formatActivity(activity) {
-	return {
+	var formattedActivity = {
```
Comment (AlexBedley) [inline demo/d2l-evaluation-hub/demo-data-parser.js]: nit: `const`

File: demo/d2l-evaluation-hub/demo-data-parser.js [hunk 2]
```diff
@@ -326,6 +327,24 @@ function formatActivity(activity) {
 			}
 		]
 	};
+
+	if (activity.isDraft) {
+		var draftEntity = {
```
Comment (AlexBedley) [inline demo/d2l-evaluation-hub/demo-data-parser.js]: nit `const`

File: demo/d2l-evaluation-hub/demo-data-parser.js [hunk 3]
```diff
@@ -263,15 +263,16 @@ function parseActivities(data, users, activityNames, courses) {
 			courseHref: getHrefForCourseId(courses.indexOf(row.courseName)),
 			activityRel: relMapping[row.activityType],
 			activityHref: getHrefForActivityNameId(activityNames.indexOf(row.activityName)),
-			localizedFormattedDate: row.localizedFormattedDate
+			localizedFormattedDate: row.localizedFormattedDate,
+			isDraft: row.isDraft
```
Comment (AlexBedley) [inline demo/d2l-evaluation-hub/demo-data-parser.js]: damn we're gonna have merge conflicts

Comment (mtjandra) [reply demo/d2l-evaluation-hub/demo-data-parser.js]: hopefully small!

File: demo/d2l-evaluation-hub/demo-data-parser.js (hunk 2 above)
Comment (mtjandra) [reply demo/d2l-evaluation-hub/demo-data-parser.js]: So is transpilation happening on local? If so I will start using const and let

**Pull request merged**